conf/messages.en

```diff
@@ -382,6 +382,24 @@ businessSettlor.startDate.error.past = The date the settlor was added to the tru
 businessSettlor.checkDetails.title = Check business settlor details
 businessSettlor.checkDetails.heading = Check business settlor details
 
+businessSettlor.countryOfResidenceYesNo.title = Do you know the settlor’s last known country of residence?
+businessSettlor.countryOfResidenceYesNo.heading = Do you know {0}’s last known country of residence?
+businessSettlor.countryOfResidenceYesNo.checkYourAnswersLabel = Do you know {0}’s last known country of residence?
+businessSettlor.countryOfResidenceYesNo.error.required = Select yes if you know the settlor’s last known country of residence
+businessSettlor.countryOfResidenceYesNo.hint = The last known country of residence is usually where the settlor lived and worked most of the time during the tax year.
+
+businessSettlor.countryOfResidenceInTheUkYesNo.title = Was the settlor a UK resident?
+businessSettlor.countryOfResidenceInTheUkYesNo.heading = Was {0} a UK resident?
+businessSettlor.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel = Was {0} a UK resident?
+businessSettlor.countryOfResidenceInTheUkYesNo.error.required = Select yes if the settlor was a UK resident
+
+businessSettlor.countryOfResidence.title = What was the settlor’s country of residence?
+businessSettlor.countryOfResidence.heading = What was {0}’s country of residence?
+businessSettlor.countryOfResidence.checkYourAnswersLabel = What was {0}’s country of residence?
+businessSettlor.countryOfResidence.error.required = Enter a country
+businessSettlor.countryOfResidence.error.length = The country of residence must be 100 characters or less
+businessSettlor.countryOfResidence.error.invalidCharacters = The country of residence must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
```

Comment:
These message keys seemingly refer to a deceased settlor (use of past tense). Are these what is in the story?

Reply:
You're probably correct. I haven't really looked at those yet. My next step was to update the navigator to plug the screens in and check the content against the ticket.